ch01.asciidoc

```diff
@@ -5,13 +5,13 @@
 === What is the Lightning Network?
 
 The Lightning Network is a payment system for Bitcoin.
-It is a second layer protocol on top of Bitcoin that defines rules and contracts which enable the fast, secure, private, trust-less and permission-less transfer of Bitcoin.
+It is a second layer protocol on top of Bitcoin that defines rules and contracts which enable the fast, secure, private, trustless and permissionless transfer of Bitcoin.
 Users of the Lightning Network are able to transfer Bitcoin among each other at virtually no cost and in real time.
-There is no necessity to wait for block confirmations for payments.
+It does not require one to wait for block confirmations for payments.
```

Comment:
I don't like the reference to `it`. the sentence before is about the Users of the Lightning Network.

Reply:
Sounds good.